src/ExtensionsSample/Program.cs

```diff
@@ -35,7 +35,7 @@ public static void Main(string[] args)
             config.UseFiles(filesConfig);
             config.UseTimers();
             config.UseSample();
-            config.UseEasyTables();
+            config.UseMobileTables();
```

Comment:
Do you guys think there will be any other Mobile artifacts that you want to bind to other than Tables? I ask because if so, it would make sense to have an extension named WebJobs.Extensions.Mobile, and config.UseMobile(). Then your extension is a package for one or more mobile bindings.